pkg/model/matrix.go

```
@@ -35,7 +35,7 @@ func (m *MatrixMessenger) SendMessage(title string, message Message, reportURL s
 	if err != nil {
 		return err
 	}
-	resp, err := http.Post(
+	resp, _ := http.Post(
```

Let's not ignore errors, return them instead